packages/angular/projects/ui-angular/src/lib/primitives-poc/phone-number/phone-number-directive/phone-number.directive.ts

```diff
@@ -0,0 +1,52 @@
+import { Directive, HostBinding, Input } from '@angular/core';
+import { nanoid } from 'nanoid';
+
+const $Directive_Selector = '[amplify-phone-number]';
```

Comment:
maybe `amplify-phone-number-input`. Later, we should abstract these out into a single `[amplify-input]` directive to remove duplciation.

Reply:
Changed.

Reply:
[075752a](https://github.com/aws-amplify/amplify-ui/pull/2420/commits/075752a0a6eb028fab66163ccc05ff6c0e625e94)